cmd/cleanup.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func CleanupCmd(fs afero.Fs) *cobra.Command {
+	cleanupCmd := &cobra.Command{
+		Use:  "cleanup",
+		Long: "Delete all fragments",
+		Args: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path, err := cmd.Flags().GetString("path")
+			if err != nil {
+				return fmt.Errorf("error parsing flag 'path': %w", err)
+			}
+
+			err = fs.RemoveAll(path)
```

Comment:
I was wondering if this implementation (where the entire folder is removed and recreated) is too greedy and can produce unexpected results. I'm thinking about the case in which users adds different files (non-YAML) in the fragments folder that get deleted by this implementation.

What do you think?